pupil/views/check/feedback.ejs

```diff
@@ -1,16 +1,19 @@
-<header class="page-header top-padding">
+
+<header class="page-header">
     <h1 class="heading-xlarge">Give feedback</h1>
 </header>
 
 <div class="grid-row">
     <div class="column-two-thirds">
+
         <% if (Object.getOwnPropertyNames(error).length > 0) { %>
-        <div class="error-summary" role="group" aria-labelledby="error-summary-heading-example-1" tabindex="-1">
+        <div class="error-summary" role="group" tabindex="-1">
```

Comment:
It was the only aria entry I found. I will cover accessibility all at once after this clean-up, the aim of this PR is having consistent, valid mark-up. If this code is still valid I will restore it.